docs/prerequisites/etcd/etcd.md

```diff
@@ -1,39 +1,37 @@
+For most use-cases it is recommended installing the Ondat etcd operator, which
+will manage creation and maintenance of Ondat's required etcd cluster. In some
+circumstances it makes sense to install etcd on separate machines outside of
+your Kubernetes cluster.
 
 ## Installing Etcd Into Your Kubernetes Cluster
 
-This is our recommended way to host etcd, in both testing and production environments.
+This is our recommended way to host etcd, in both testing and production
+environments.
 
 ## Configuring Storage for Etcd
 
-We highly recommend using cloud provider network attached disks for storing etcd data, such as EBS volumes, Google Persistent Disks, Azure Disks, etc. This allows the etcd operator to recover from node failures.
+We highly recommend using cloud provider network attached disks for storing
+etcd data, such as EBS volumes, Google Persistent Disks, Azure Disks, etc. This
+allows the etcd operator to recover from node failures.
 
-For testing environments a node-local storage option can be used, such as [Local Path Provisioner](https://github.com/rancher/local-path-provisioner). This will store etcd data on the node hosting an etcd pod
+For testing environments a node-local storage option can be used, such as
+[Local Path Provisioner](https://github.com/rancher/local-path-provisioner).
+This will store etcd data on the node hosting an etcd pod
 
-> ⚠️ The `local-path` StorageClass is only recommended for **non production** clusters, as this stores all the data of the `etcd` peers locally, which makes it susceptible to state being lost on node failures.
-
-### How to set up an EBS CSI Driver
-
-In AWS, you can use EBS volumes to host the etcd PVCs. The Ondat etcd usage of disk depends on the size of the Kubernetes cluster. However, it is recommended that the disks have at least 800 IOPS at any point in time. The best cost effective storage class that fulfils such requirements is gp3. If gp2 is used, it is paramount to use a volume bigger than 256Gi as it will have enough IOPS even when the burstable credits are exhausted.
-
-To use a gp3 storage class in Kubernetes it is required to install the Amazon CSI Driver. Follow [this guide](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) to install. The procedure is comprehended by the following steps:
-
-* Create IAM permissions <https://docs.aws.amazon.com/eks/latest/userguide/csi-iam-role.html>
-* Install the CSI driver
-  * [Using EKS addon](https://docs.aws.amazon.com/eks/latest/userguide/managing-ebs-csi.html)
-  * [Using self-managed add on](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/install.md) (AWS clusters, but not in EKS)
+> ⚠️ The `local-path` StorageClass is only recommended for **non production**
+> clusters, as this stores all the data of the `etcd` peers locally, which
+> makes it susceptible to state being lost on node failures.
```

Comment:
```suggestion
> makes it susceptible to state being lost on node failures.
> In the case of `local-path` storage, a minimum of 5 etcd nodes
> is recommended to increase resilience.
```

Reply:
FYI it's recommended to run 5 nodes even when using EBS (or other cloud disks)